can_tools/scrapers/official/VA/va_vaccine.py

```diff
@@ -81,3 +78,288 @@ def normalize(self, data) -> pd.DataFrame:
         df["dt"] = self._retrieve_dt()
         df["vintage"] = self._retrieve_vintage()
         return df.drop(["variable"], axis="columns")
+
+
+class VirginiaCountyVaccineDemographics(VirginiaVaccine):
+    viewPath = "VirginiaCOVID-19Dashboard-VaccineDemographics/VirginiaCOVID-19VaccineDemographics"
+    filterFunctionName = (
+        "[federated.1a55gj51go17vw1cds8i50u4b8by].[none:Locality Name:nk]"
+    )
+    filterFunctionValue = None
+    has_location = False
+    location_type = "county"
+    counties = [
```

Comment:
fly by comment.... Seems like we should be able to pull this from the CSV files in bootstrap_data instead of listing them out here. Would it be worth the effort to do that?

Reply:
This should/might work for this: 
```
import os.path
counties = list(
            pd.read_csv(
                os.path.dirname(__file__) + "/../../../bootstrap_data/locations.csv"
            ).query("state == @self.state_fips and name != 'Virginia'")["name"]
        )
```

Reply:
@sglyon, does the same file structure exist on the server? If so, then @smcclure17 code is good.